1 table fpmembers.sql

```diff
@@ -0,0 +1,37 @@
+use FitPlexDB
+go 
+drop table if exists Member
+go 
+create table dbo.Member(
+     MemberID int not null primary key identity,
+     MemberFirstName varchar(25) not null
+          constraint c_Member_first_name_cannot_be_blank check(MemberFirstName <> ''),
+     MemberLastName varchar(25) not null 
+          constraint c_Member_last_name_cannot_be_blank check(MemberLastName <> ''),
+     MemberPhoneNum char(8) not null
+          constraint c_Member_phone_number_must_have_seven_digits check(MemberPhoneNum like '[1-9][0-9][0-9][-][0-9][0-9][0-9][0-9]'),
+          constraint u_Member_phone_number unique(MemberPhoneNum),
+     MemberInitialWeightLb tinyint not null
```

Comment:
Tinyint only goes to 255. There are people that weigh more then that. The same problem applies in the next 2 columns.